packages/kit/src/core/build/build_client.js

```diff
@@ -29,6 +29,10 @@ export async function build_client({
 	output_dir,
 	client_entry_file
 }) {
+	process.env.VITE_APP_VERSION = config.kit.version?.name || Date.now().toString();
+	process.env.VITE_APP_VERSION_FILE = `${config.kit.appDir}/version.json`;
+	process.env.VITE_APP_VERSION_POLL_INTERVAL = `${config.kit.version?.pollInterval || 60_000}`;
```

Comment:
Config is always fully defined — defaults should be specified in `options.js` (I realise this might make testing slightly more finicky)

```suggestion
	process.env.VITE_APP_VERSION = config.kit.version.name;
	process.env.VITE_APP_VERSION_FILE = `${config.kit.appDir}/version.json`;
	process.env.VITE_APP_VERSION_POLL_INTERVAL = `${config.kit.version.pollInterval}`;
```